packages/next/src/server/app-render/encryption.ts

```diff
@@ -109,11 +113,39 @@ export async function encryptActionBoundArgs(actionId: string, args: any[]) {
     throw error
   }
 
+  const workUnitStore = workUnitAsyncStorage.getStore()
```

Comment:
Maybe it would simplify the code a bit just to write:

```ts
if (!workUnitStore) {
  return encodeActionBoundArg(actionId, serialized)
}
```

Then the remaining code can remove the extra `?` and ternaries.